README.yaml

```diff
@@ -0,0 +1,131 @@
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+  - name: "Discourse Forum"
+    image: "https://img.shields.io/discourse/https/ask.sweetops.com/posts.svg"
+    url: "https://ask.sweetops.com/"
+
+# List any related terraform modules that this module may be used with or that this module depends on.
+related:
+  - name: "terraform-null-label"
+    description: "Terraform module designed to generate consistent names and tags for resources. Use terraform-null-label to implement a strict naming convention."
+    url: "https://github.com/cloudposse/terraform-null-label"
+
+# List any resources helpful for someone to get started. For example, link to the hashicorp documentation or AWS documentation.
+references:
+  - name: "Terraform Standard Module Structure"
+    description: "HashiCorp's standard module structure is a file and directory layout we recommend for reusable modules distributed in separate repositories."
+    url: "https://www.terraform.io/docs/modules/index.html#standard-module-structure"
+  - name: "Terraform Module Requirements"
+    description: "HashiCorp's guidance on all the requirements for publishing a module. Meeting the requirements for publishing a module is extremely easy."
+    url: "https://www.terraform.io/docs/registry/modules/publish.html#requirements"
+  - name: "Terraform `random_integer` Resource"
+    description: "The resource random_integer generates random values from a given range, described by the min and max attributes of a given resource."
+    url: "https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/integer"
+  - name: "Terraform Version Pinning"
+    description: "The required_version setting can be used to constrain which versions of the Terraform CLI can be used with your configuration"
+    url: "https://www.terraform.io/docs/configuration/terraform.html#specifying-a-required-terraform-version"
+
+# Short description of this project
+description: |-
+  This `terraform-aws-eks-iam-role` project provides a simplified mechanism for provisioning
+  [AWS EKS Service Account IAM roles](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+# Introduction to the project
+#introduction: |-
+#  This is an introduction.
+
+# How to use this module. Should be an easy example to copy and paste.
+usage: |-
+  Here's how to invoke this example module in your projects
+
+  ```hcl
+  module "eks_iam_role" {
+    source = "https://github.com/cloudposse/terraform-aws-eks-iam-role.git?ref=master"
+
+    namespace   = var.namespace
+    environment = var.environment
+    stage       = var.stage
+    name        = var.name
+    delimiter   = var.delimiter
+    attributes  = var.attributes
+    tags        = var.tags
+
+    aws_account_number          = local.account_id
+    eks_cluster_oidc_issuer_url = module.eks_cluster.eks_cluster_identity_oidc_issuer
```

Comment:
do we need to remove `https://` from the issue URL (I don't remember)

Reply:
Yes, but the module does that